### DISCUSSION TITLE
Ideas for AzuraCast Features

### DISCUSSION THREAD

**Discussion by M0rzze** [AzuraCast/AzuraCast#7552] [Ideas]
hi, I'm new here, maybe I just didn't find something somewhere, or I don't understand something, don't judge me too harshly

1) Customizable Interval for Jingles:
Description: Currently, AzuraCast allows setting a fixed interval, like every 3 tracks or every 15 minutes. It would be useful to have a random interval option for jingles, for example, "every 3–5 tracks" for a more dynamic broadcast experience.
Possible Implementation: Add a parameter for random playback interval in the jingle settings.

2) Option to Play a Specific File Before Tracks in Certain Playlists:
Description: The ability to play a specific file (like a jingle or announcement) before tracks from specific playlists would be useful. For instance, for a “News” playlist, a brief announcement could play before each track.
Possible Implementation: An option in playlist settings to automatically queue a specific file before a track begins.

3) Track “Like” Feature for Listeners:
Description: Adding a “like” button for listeners to show appreciation for specific tracks would help in tracking popular songs, which could influence rotation decisions.
Possible Implementation: A like button in the player interface that either displays the total likes or stores like data on the server. (I don't quite understand why this might be needed, perhaps to compile some kind of chart, but there was such a request)

**Comment by BusterNeece** [comment]
@M0rzze Hello and thank you for your submission! We actually have a special site just for tracking feature requests in a way that makes it easy for us to keep track of which ones are the most popular (and lets our community members vote on which ones they want to see most): https://features.azuracast.com/

I'd recommend submitting these items there, though you should search the existing feature requests first, as I suspect many of these are already requested on there.

**Reply by M0rzze** [reply]
Thank you very much, for some reason I missed this special site. I will go study it in detail
